frontend/src/components/c-ramp.vue

```diff
@@ -171,21 +171,21 @@ export default defineComponent({
     },
 
     // position for commit granularity
-    getCommitPos(i: number, total: number) {
+    getCommitPos(i: number, total: number): number {
       return (((total - i - 1) * window.DAY_IN_MS) / total)
           / (this.getTotalForPos(this.sdate, this.udate) + window.DAY_IN_MS);
     },
     // position for day granularity
-    getSlicePos(date: string) {
+    getSlicePos(date: string): number {
       const total = this.getTotalForPos(this.sdate, this.udate);
       return (new Date(this.udate).valueOf() - new Date(date).valueOf()) / (total + window.DAY_IN_MS);
     },
 
     // get duration in miliseconds between 2 date
-    getTotalForPos(sinceDate: string, untilDate: string) {
+    getTotalForPos(sinceDate: string, untilDate: string): number {
       return new Date(untilDate).valueOf() - new Date(sinceDate).valueOf();
     },
-    getRampColor(commit: CommitResult, slice: Commit) {
+    getRampColor(commit: CommitResult, slice: Commit): number | '-deletes' {
```

Comment:
We could add a return type for `getSliceColor()` on line 194 as well.

Reply:
Fixed! Thank you